woocommerce/woocommerce-ios
# [Product Multi-Selection] Prepare for release

Closes: #8888 

## Description
This PR switches the `productMultiSelectionM1` feature flag to `true`, so Product Multi-Selection is available in release builds.

## Testing instructions
- Run the app
- Go to Orders > Tap `+` > `+ Add Products` > See that products can be tapped on a multi-selection basis

---
